designer/client/src/stylesheets/editor.scss

```diff
@@ -245,6 +247,20 @@ p code {
       content: "⟩";
     }
   }
+
+  .fileUploadIconContainer {
```

Comment:
Can we stick with [BEM](https://getbem.com) for naming conventions?

```suggestion
  .file-upload {
```

Reply:
Done